src/wmtk/TriMesh.hpp

```diff
@@ -16,6 +16,7 @@ class TriMesh : public Mesh
     TriMesh& operator=(const TriMesh& o);
     TriMesh& operator=(TriMesh&& o);
 
+    PrimitiveType top_simplex_type() const override { return PrimitiveType::Face; }
```

Comment:
i have a preference for shoving all of these implementations in to cpp files. makes headers cleaner, but not everyone thinks this way